README.md

```diff
@@ -42,7 +42,7 @@ $ python3 -m unittest tests.test_symbol_factory.TestPhone
 
 ```python
 # Place this line in the place were you want to start to debug.
-import ipdb; ipdb.set_trace()
+import ipdb; ipdb.set_trace() # break point
```

Comment:
@carlosvivasb  and @Anahis259 , this is just to illustrate that that line of code is used to make a breakpoint in the code.

Reply:
good

Reply:
Ok